features/geocoder/nominatim/src/main/java/org/opennms/features/geocoder/nominatim/NominatimGeocoderService.java

```diff
@@ -55,12 +55,16 @@
     private String m_emailAddress;
     private String m_referer;
 
+    private boolean m_useSystemProxy = true; // true == legacy behaviour
```

Comment:
Similar situation here: I believe it should be `m_useSystemProxy = false; // false == legacy behaviour.`

Reply:
see my comment above. Here it would be line 63. Please let me know if I should change it to false. Thanks!